src/drm/drm_handler.js

```diff
@@ -3,6 +3,18 @@ import { Log } from '@clappr/core'
 const MESSAGE_TYPE = 'application/vnd.ms-playready.initiator+xml'
 const DRM_SYSTEM_ID = 'urn:dvb:casystemid:19219'
 
+export const getFullChallengeMessageTemplate = header => {
+  const template = '<?xml version="1.0" encoding="utf-8"?>'
+  + '<PlayReadyInitiator xmlns="http://schemas.microsoft.com/DRM/2007/03/protocols/">'
+  + '<LicenseAcquisition>'
+  + '<Header>'
+  + `${header}`
+  + '</Header>'
```

Comment:
We could add the `content` (which is the video URL) and template strings here 
```suggestion
  + '</Header>'
  + '<Content>${content}</Content>'
```